libs/feature/editor/src/lib/components/record-form/record-form.component.ts

```diff
@@ -8,19 +8,21 @@ import {
   EditorFieldWithValue,
   EditorSectionWithValues,
 } from '../../+state/editor.models'
+import { getGlobalConfig } from '@geonetwork-ui/util/app-config'
+import { UiInputsModule } from '@geonetwork-ui/ui/inputs'
```

Comment:
What do you need these for?

Reply:
I need getGlobalConfig to get the list of open-data licenses, stored in the config file.
And I need UIInputsModule because the switch component is in that lib (I think? It could also be leftover from v1 of the PR, will check)

Reply:
Just so you know, all new components are implemented as standalone and imported directly. We try not to import Angular modules anymore in this project.

As for the licenses configuration, it could be part of the license field configuration in `libs/feature/editor/src/lib/fields.config.ts`, but I'll let the final word to @jahow.

Reply:
Indeed it should go into fields.config.ts! thanks!

Reply:
Noted, thanks :slightly_smiling_face: